assessments/FastBridge_English/earthmover.yaml

```diff
@@ -0,0 +1,2015 @@
+version: 2
+config:
+  log_level: INFO
+  output_dir: ${OUTPUT_DIR}
+  memory_limit: 1GB
+  state_file: ${STATE_FILE}
+  show_graph: false
+  show_stacktrace: true
+  parameter_defaults:
+    # You can set these values at the command line or leave the defaults
+    OUTPUT_DIR: ./output/
+    STATE_FILE: ./tmp/.earthmover/runs.csv
+    API_YEAR: ''
+    STUDENT_ID_NAME: 'edFi_studentUniqueID' # default to the column added by the apply_xwalk package of student ID xwalking feature
+    POSSIBLE_STUDENT_ID_COLUMNS: StateID,LocalID
+    DESCRIPTOR_NAMESPACE_OVERRIDE: uri://ed-fi.org
+
+sources:
+  {% set assessment_periods = [
+    'Fall',
+    'Winter',
+    'Spring',
+    'ScreeningPeriod4',
+    'ScreeningPeriod5'
+    ] %}
+  
+  {% set objective_names = [
+    'EarlyReadingEnglish',
+    'ConceptsofPrint',
+    'DecodableWords',
+    'LetterNames',
+    'LetterSounds',
+    'NonsenseWords',
+    'OnsetSounds',
+    'OralRepetition',
+    'WordRhyming',
+    'SentenceReading',
+    'SightWords',
+    'WordBlending',
+    'WordSegmenting',
+    'CBMR-English'
+    ] %}
+
+  input:
+    file: ${INPUT_FILE}
+    header_rows: 1
+    columns:
+      - Assessment

```

Comment:
are you including these here to overwrite the column names? It is otherwise unnecessary to list them all out